src/js/actions/event/data.js

```diff
@@ -91,7 +91,7 @@ export const deleteEventFailure = error => ({
   error
 });
 
-export function getEvent (token, event_id) {
+export function getEvent (token, event_id, nav) {
```

Comment:
Can we refer to `nav` as `navigation`, just so it's 100% unambiguous? `nav` could mean navbar, navigation, navigator...